main/repl/timerServices.md

```diff
@@ -133,71 +128,62 @@ history[4] 1632170402207n
-times that are a multiple of the specified interval following `baseTime`. `interval`
-is the delay between times when `wake()` is called. Since block times are coarse-grained,
-the actual call may occur later, but this won't change when the
-next event will be called. 
+times that are a multiple of the specified `interval`, with the first update happening
+after the specified `delay` has elapsed. Since block times are coarse-grained,
+the actual calls when using `chainTimerService` may occur less frequently than the specified
+`interval`.
 ```js
-command[6] E(home.localTimerService).makeRepeater(5_000n,10_000n)
+command[6] E(home.localTimerService).makeRepeater(5_000n, 10_000n)
 history[6] [Alleged: presence o-124]{}
 ```
 
 A longer example of creating and using Repeaters:
 
-```
-command[3]  makeHandler = () => { let calls = 0; const args = []; return { getCalls() {
-return calls; }, getArgs() { return args; }, wake(arg) { args.push(arg); calls += 1; }, }; }
-history[3]  [Function makeHandler]
-command[4]  h1 = makeHandler()
-history[4]  {"getCalls":[Function getCalls],"getArgs":[Function getArgs],"wake":[Function wake]}
-command[5]  h2 = makeHandler()
-history[5]  {"getCalls":[Function getCalls],"getArgs":[Function getArgs],"wake":[Function wake]}
+```js
+command[3]  makeWakeCounter = () => {
+  let count = 0;
+  return Far('counter', { getCount() { return count; }, wake(_t) { count += 1; }, });
+}
+history[3]  [Function makeWakeCounter]
+command[4]  c1 = makeWakeCounter()
+history[4]  [Object Alleged: counter]{"getCount":[Function getCount],"wake":[Function wake]}
+command[5]  c2 = makeWakeCounter()
+history[5]  [Object Alleged: counter]{"getCount":[Function getCount],"wake":[Function wake]}
 command[6]  tl = home.localTimerService
-history[6]  [Presence o-59]  
+history[6]  [Object Alleged: timerService]{}
 command[7]  tc = home.chainTimerService
-history[7]  [Presence o-55]  
-command[8]  rl = E(tl).makeRepeater(7, 1500)
-history[8]  [Presence o-64]  
-command[9]  rc = E(tc).makeRepeater(7, 1)
-history[9]  [Presence o-65]  
-command[10]  E(rl).schedule(h1)
+history[7]  [Object Alleged: timerService]{}
+command[8]  rl = E(tl).makeRepeater(7n, 1500n)
+history[8]  [Object Alleged: vatRepeater]{}
+command[9]  rc = E(tc).makeRepeater(7n, 15n)
+history[9]  [Object Alleged: vatRepeater]{}
+command[10]  E(rl).schedule(c1)
 history[10]  1571783040007n
-command[11]  E(rc).schedule(h2)
+command[11]  E(rc).schedule(c2)
 history[11]  1571783051n
-command[12]  h1.getCalls()
-history[12]  3
-command[13]  h2.getCalls()
-history[13]  1
 ...
-command[22]  h1.getCalls()
-history[22]  50
-command[23]  h1.getCalls()
-history[23]  53
-command[24]  h1.getCalls()
-history[24]  54
-command[25]  E(tl).getCurrentTimestamp()
-history[25]  1571783375000n
-command[26]  E(tc).getCurrentTimestamp()
-history[26]  1571783384n
+command[22]  Promise.resolve(E(tl).getCurrentTimestamp()).then(t => [t, c1.getCount()])
```

Comment:
Why add `Promise.resolve()`?

Reply:
In case it was a mistake for `getCurrentTimestamp` to return a promise rather than a bigint. If we are committed to the former, then I can drop this `Promise.resolve`.

Reply:
Which I see on reinspection is necessarily the case because that's how `E()` works. Fixing now.